eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php

```diff
@@ -1457,6 +1457,31 @@ static function (Connection $connection) use ($nestedFolderLocation) {
         $urlAliasService->refreshSystemUrlAliasesForLocation($nestedFolderLocation);
     }
 
+    public function testOverrideHistoryUrlAliasAtTheSameLocation(): void
+    {
+        $repository = $this->getRepository();
+        $urlAliasService = $repository->getURLAliasService();
+        $locationService = $repository->getLocationService();
+
+        $folderNames = ['eng-GB' => 'foo'];
+        $folder = $this->createFolder($folderNames, 2);
+
+        $location = $locationService->loadLocation($folder->contentInfo->mainLocationId);
+        $newLocation = $locationService->loadLocation(43);
```

Comment:
Can we improve here independence of the tests by making new content/location on the fly instead of hardcoding database ID?